### PR TITLE
Revise census metrics to reflect current reality

### PR DESCRIPTION
**Pull request by wlach**
* Move legacy census metrics (ADI, legacy DAU definitin) to historical
  section
* Move coverage ping documentation to pings sections

### CODE REVIEW

File: src/concepts/censuses.md [hunk 1]
```diff
@@ -65,6 +66,8 @@ age, or update failures.
 
 ### DAU / Daily Active Users
 
+> **⚠** The mechanism described below is no longer used: see the [DAU definition in metrics](../metrics/metrics.md#daily-active-users-dau) for the currently-accepted definition.
```
Comment (tdsmith) [inline src/concepts/censuses.md:69]: I don't think this has changed.

File: src/concepts/history.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 # A brief history of Firefox data collection
 
 > This section was originally included in the [Project Smoot existing metrics report][smootv1]
-(Mozilla internal link).
+(Mozilla internal link), but has since been extensively modified.
```
Comment (tdsmith) [inline src/concepts/history.md:4]:
```suggestion
(Mozilla internal link); the DTMO version has been updated to reflect changes to the data platform.
```

File: src/concepts/censuses.md [hunk 3]
```diff
@@ -1,14 +1,15 @@
-## Censuses
+## Legacy Census Metrics
```
Comment (ecsmyth) [inline src/concepts/censuses.md:1]: Should this section also include the definitions for aDAU and aMAU?

File: src/concepts/history.md [hunk 4]
```diff
@@ -30,12 +30,11 @@ Daily Users, which was [renamed to Active Daily
 Installs](https://bugzilla.mozilla.org/show_bug.cgi?id=812282) (ADI) by
 2012.
 
-Work is underway to replace `blocklist.xml` with a [Remote
+As of August 2020, this mechanism has been superseded by a [Remote
```
Comment (ecsmyth) [inline src/concepts/history.md:33]: Could change this to Fx72, which is when the staged rollout of the remote settings-based implementation started, or Fx73 when the new implementation became the default.

File: src/concepts/censuses.md (hunk 3 above)
Comment (tdsmith) [reply src/concepts/censuses.md:1]: fwiw there's some internal commentary at https://mozilla-private.report/smoot-existing-metrics/book/20_metric_correlations.html#dau-and-dau-5-uris; it's probably good to have an aDAU grave marker on DTMO and I'll toss that on my todo list.

File: src/concepts/censuses.md (hunk 1 above)
Comment (wlach) [reply src/concepts/censuses.md:69]: There's overlap between this and the definition in metrics for sure, but I think we need to make it clear that this isn't the currently accepted one. I'm open to changing the wording of this warning, but I think we should maintain it.

Comment (tdsmith) [reply src/concepts/censuses.md:69]:
```suggestion
> **⚠** This description of DAU is not authoritative; please see the [DAU definition in metrics](../metrics/metrics.md#daily-active-users-dau) for the canonical definition.
```

Comment (wlach) [inline src/concepts/censuses.md:69]: How about:

```suggestion
> **⚠** This is out of date. See the [DAU definition in metrics](../metrics/metrics.md#daily-active-users-dau) for the currently-accepted version.
```

File: src/concepts/history.md (hunk 4 above)
Comment (wlach) [inline src/concepts/history.md:33]: @emsyth wdyt?

```suggestion
As of Firefox 73 (released February 11, 2020), this mechanism has been superseded by a [Remote
```

Comment (wlach) [reply src/concepts/history.md:33]: Sounds good, I think just referencing 73 is sufficient context.

File: src/concepts/censuses.md (hunk 3 above)
Comment (wlach) [reply src/concepts/censuses.md:1]: Yeah, moving https://docs.telemetry.mozilla.org/cookbooks/active_dau.html to the historical section is also on my list of things to do (of course I wouldn't object if @tdsmith did it for me!)

File: src/concepts/history.md (hunk 4 above)
Comment (ecsmyth) [reply src/concepts/history.md:33]: lgtm

**Pull request merged**